<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>617</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AcadEref</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Student</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableView" name="tableStudent"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Teacher</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="tableTeacher"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Counsellor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableView" name="tableCounsellor"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Course</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableView" name="tableCourse"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Account</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTableView" name="tableAccount"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionStudent"/>
     <addaction name="actionTeacher"/>
     <addaction name="actionCounsellor"/>
     <addaction name="actionCourse"/>
     <addaction name="actionAccount"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuAdd"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStudent">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="actionTeacher">
   <property name="text">
    <string>Teacher</string>
   </property>
  </action>
  <action name="actionCounsellor">
   <property name="text">
    <string>Counsellor</string>
   </property>
  </action>
  <action name="actionCourse">
   <property name="text">
    <string>Course</string>
   </property>
  </action>
  <action name="actionStudent_2">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="actionTeacher_2">
   <property name="text">
    <string>Teacher</string>
   </property>
  </action>
  <action name="actionCounsellor_2">
   <property name="text">
    <string>Counsellor</string>
   </property>
  </action>
  <action name="actionCourse_2">
   <property name="text">
    <string>Course</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="deleteStudent">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="deleteTeacher">
   <property name="text">
    <string>Teacher</string>
   </property>
  </action>
  <action name="deleteCounsellor">
   <property name="text">
    <string>Counsellor</string>
   </property>
  </action>
  <action name="deleteCourse">
   <property name="text">
    <string>Course</string>
   </property>
  </action>
  <action name="deleteAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
